evap/evaluation/tools.py

```diff
@@ -141,7 +144,7 @@ def calculate_results(course):
     if course.state != "published":
         return _calculate_results_impl(course)
 
-    cache_key = str.format('evap.staff.results.tools.calculate_results-{:d}', course.id)
+    cache_key = 'evap.staff.results.tools.calculate_results-%d' % course.id
```

Comment:
i mean either leave it like it was or do `'foo'.format(...)` if that helps, but don't use the percent operator.